README.md

```diff
@@ -41,6 +41,15 @@ So far, I've added features that allow you to sneak source code into a machine u
     from matplotlib_visual.piechart import display
     display("<filename-without-extension>")
 ```
+
+- **New Feature: Web Scraping**
```

Comment:
Maybe new this line, write it somethin like "Scraping content from cl1p.net" so that it looks linear with the rest of the readme

Reply:
okay, i will redo the comment.